.env.example

```
@@ -18,3 +18,9 @@ GOOGLE_CLIENT_SECRET=your_google_client_secret_here
 GOOGLE_REDIRECT_URI=http://localhost:5173/oauth/callback
 GOOGLE_REFRESH_TOKEN=your_google_refresh_token_here
 GOOGLE_CALENDAR_ID=primary
+
+# Configuration Email (Resend)
+# Pour activer l'envoi d'emails de confirmation automatiques
+# Inscrivez-vous sur https://resend.com et obtenez une clé API
+RESEND_API_KEY=re_9tQBWc3R_FW1eBULbk2igSshem5z9wpq8
+APP_URL=https://webapp-frtjapec0-ikips-projects.vercel.app
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APP_URL key should go before the RESEND_API_KEY key

(UnorderedKey)